src/cli/context.rs

```diff
@@ -140,7 +144,11 @@ impl CliContext {
     }
 
     pub fn staging_path(&self) -> PathBuf {
-        self.project_dir.join(RV_DIR_NAME).join(STAGING_DIR_NAME)
+        if self.library.custom {
+            self.project_dir.join(STAGING_DIR_NAME)
```

Comment:
would this then put staging inside the project dir? - what happens if someone has a folder named staging for just general usage... 😬

Reply:
can do